games-action/cylindrix/cylindrix-1.0.recipe

```diff
@@ -1,19 +1,57 @@
+SUMMARY="A combat game"
 DESCRIPTION="Cylindrix is a 3 degrees of freedom combat game"
 HOMEPAGE="http://www.hardgeus.com/cylindrix/"
-SOURCE_URI="http://www.hardgeus.com/cylindrix/cylindrix-1.0.tar.gz"
+COPYRIGHT="2001 John R. McCawley III, Anthony Thibault"
+LICENSE="GNU GPL v2"
 REVISION="1"
-CHECKSUM_MD5="914cf70b0a95f4a36cfdfde7edbaa240"
-STATUS_HAIKU="stable"
-DEPEND="media-libs/allegro >= 4.4.1.1"
+SOURCE_URI="http://pkgs.fedoraproject.org/lookaside/pkgs/cylindrix/cylindrix-1.0.tar.gz/914cf70b0a95f4a36cfdfde7edbaa240/cylindrix-1.0.tar.gz"
+CHECKSUM_SHA256="33bfc264d906a1977090e47a38f544551d2939d4c0ea01e1792f63ab4fa06f20"
+
+SOURCE_DIR="cylindrix"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cylindrix
+	cmd:cylindrix$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:liballeg$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:liballeg$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:cmp
+	cmd:aclocal
+	cmd:automake
+	cmd:autoheader
+	cmd:libtoolize
+	cmd:makeinfo
+	"
+
 BUILD()
 {
-	cd cylindrix
-	CFLAGS='-O2' ./configure --prefix=`finddir B_COMMON_DIRECTORY`
-	make
+	libtoolize -fci
+	aclocal -I $sourceDir
+	autoconf
+	automake --add-missing
+    runConfigure --omit-dirs binDir ./configure --bindir=$appsDir/Cylindrix
+	make $jobArgs
 }
 
 INSTALL()
 {
-	cd  cylindrix
 	make install
+	mv $appsDir/Cylindrix/cylindrix $appsDir/Cylindrix/Cylindrix
+	cd $sourceDir
+	cp -r 3d_data gamedata pcx_data stats wav_data $appsDir/Cylindrix
```

Comment:
**`make install`** should normaly take care of installing **`3d_data gamedata pcx_data stats wav_data`** to the appropriate places, under $dataDir, I guess, so you should not need to copy them.
Hmm, **`stats`** should go to a writable directory. Does configure have an option to put it in a separate directory tree?